bin/pycbc_optimize_snr

```diff
@@ -353,12 +354,28 @@ assert maxchirp < 250.
 maxm1 = cv._mass2_from_mchirp_mass1(maxchirp, MIN_CPT_MASS)
 mineta = max(cv.eta_from_mass1_mass2(maxm1, MIN_CPT_MASS), 0.01)
 
-# Boundaries of the optimization space: dict of (min, max) tuples
+# Use max chirp and min eta to find the upper bounds on mass1 and mass2
+#  If masses can be > 3 then allow them BH spin range
+maxmass1 = cv.mass1_from_mchirp_eta(maxchirp, mineta)
```

Comment:
Is maxmass1 different to maxm1? maybe add a comment stating how

Reply:
These are the same value, I will keep the original one + the new maxmass2.